src/asyncOperationManagerUtils.js

```diff
@@ -95,10 +96,14 @@ const getAsyncOperation = (
 const shouldRunOperation = (descriptorId, params) => {
   const state = asyncOperationManagerState.getState();
 
+  // @TODO: createAsyncOperationAction functions in reduxIntegration need to have another argument to specify AOM parameters
+  // so we can place them in a params property on the action
+  const actualParams = has(params, 'params') ? params.params : params;
```

Comment:
I'm having trouble picturing the usage. Can you reply with an example or put it in the description?